qemu/tests/cfg/win_sigverif.cfg

```diff
@@ -7,17 +7,25 @@
     sigverif_log = "c:\Users\Public\Documents\SIGVERIF.TXT"
     check_sigverif_cmd = 'type ${sigverif_log} | findstr /i "%s"'
     clean_sigverif_cmd = "del ${sigverif_log}"
+    variants:
+        - @default:
+        - msft_sign_check:
+          type = win_msft_sign_check
+          vio_driver_chk_cmd = 'driverquery /si | find "%s"'
```

Comment:
It's better to ignores the case of the characters when searching for the string with /i in case the device name is changed with that.

Reply:
Modified, thanks